FWCore/Concurrency/scripts/edmStreamStallGrapher.py

```diff
@@ -424,7 +424,7 @@ def printStalledModulesInOrder(stalledModules):
 
     def sumSort(i,j):
         return cmp(i[1],j[1])
-    priorities.sort(cmp=sumSort, reverse=True)
+    priorities.sort(key=cmp_to_key(sumSort), reverse=True)
```

Comment:
```suggestion
    priorities.sort(key=lambda a: a[1], reverse=True)
```
And then delete the function `sumSort`. This allows changing the (possibly large) list inplace which is probably more efficient.